mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
or I guess since you want it sorted it would be:
```
peaks = np.sort(argrelmax(gfp, order=order)[:10])
```
I find that a lot easier to understand, assuming it does the same thing.